src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
```

Comment:
It is not a good practice to have empty divs in the code. Besides, the logo should be clickable.
Please refactor it to something like this:
```suggestion
      <div class="header__logo">
        <a href="/">
          <img src="path/to/logo/image" alt="logo" />
        </a>
      </div>
```
Adjust the styles accordingly.